src/legacy/core_plugins/expressions/public/np_ready/public/loader.ts

```diff
@@ -57,6 +63,12 @@ export class ExpressionLoader {
       this.render(data);
     });
 
+    this.params = {
+      searchContext: { type: 'kibana_context' },
+      extraHandlers: params.extraHandlers,
+    };
```

Comment:
I don't really understand why you've added this

Reply:
sorry, some context: we will use this for all core visualizations, and to make maps work for now we will still need to pass in some additional information.